src/style.css

```diff
@@ -1 +1,57 @@
 /* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  position: relative;
+  width: 96px;
+  top: 22px;
+  left: 22px;
```

Comment:
Don't fix container size (if there is no such a requirement). Setting a fixed width may cause issues with responsiveness. Consider using max-width or a more responsive size unit like percentages or viewport units if needed.